scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl

```diff
@@ -801,6 +822,75 @@ task SubpopulationAlleleNumber {
     }
 }
 
+task DuplicateAnnotations {
+    input {
+        String query_project_id
+        String fq_vat_table
+        String last_modified_timestamp
+    }
+    # there are vids with duplicate annotations
+    # make sure there aren't matching sites with mis-matched ANs and matching variants with mis-matched ACs
+
+    String pf_file = "pf.txt"
+    String results_file = "results.txt"
+
+    command <<<
+        # Prepend date, time and pwd to xtrace log entries.
+        PS4='\D{+%F %T} \w $ '
+        set -o errexit -o nounset -o pipefail -o xtrace
+
+        echo "project_id = ~{query_project_id}" > ~/.bigqueryrc
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv '
+        SELECT contig, position, gvs_all_an, COUNT(DISTINCT gvs_all_an) AS an_count
+        FROM `~{fq_vat_table}`
+        GROUP BY contig, position, gvs_all_an
+        HAVING an_count > 1
+        ' > bq_an_output.csv
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv '
+        SELECT contig, position, gvs_all_ac, COUNT(DISTINCT gvs_all_ac) AS ac_count
+        FROM `~{fq_vat_table}`
+        GROUP BY contig, position, gvs_all_ac
+        HAVING ac_count > 1
+        ' > bq_ac_output.csv
+
+        # get number of lines in bq query output
+        NUMANRESULTS=$(awk 'END{print NR}' bq_an_output.csv)
+        NUMACRESULTS=$(awk 'END{print NR}' bq_ac_output.csv)
+
+
+        echo "false" > ~{pf_file}
+        # if the results of the queries have any rows, that means there are sites with mis-matched gvs_all_an or gvs_all_ac
+        if [[ $NUMANRESULTS != "0" ]]; then
+          echo "The VAT table ~{fq_vat_table} has mis-matched calculations for AC, and AC of subpopulations" > ~{results_file}
+          cp bq_an_output.csv ~{results_file}
+        elif [[ $NUMACRESULTS != "0" ]]; then
+          echo "The VAT table ~{fq_vat_table} has mis-matched calculations for AN, and AN of subpopulations" > ~{results_file}
+          cp bq_ac_output.csv ~{results_file}
+        else
+          echo "The VAT table ~{fq_vat_table} has correct calculations for AN, AC, AN of subpopulations and AC of subpopulations" > ~{results_file}
+          echo "true" > ~{pf_file}
+        fi
+
+    >>>
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:423.0.0"
```

Comment:
I would prefer `alpine` over `slim` over the implicit kitchen-sink version, unless either of those don't work for some surprising reason.

```suggestion
        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:423.0.0-alpine"
```